lib/game.rb

```diff
@@ -0,0 +1,115 @@
+require 'pry'
+
+class Game 
+  attr_accessor :player_1, :player_2, :board
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ]
+
+
+
+  def initialize(player_1 = Player::Human.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+
+  def current_player
+    board.turn_count % 2 == 0 ? player_1 : player_2
+  end
+
+  def over?
+    draw? || won? ? true : false
```

Comment:
Just like with the `if` statement above, a ternary operator with an explicit `true`/`false` is extraneous.
```
def over?
  draw? || won?
end
```